### DISCUSSION TITLE
Easy way to run applications that don't need a port open

### DISCUSSION THREAD

**Discussion by aidendotgg** [coollabsio/coolify#4102] [Feature Requests]
There should be an easy way to make it so you can run an application without allocating a port to it. Basically just an option for any application that doesn't need a port open like a discord bot.

**Comment by andrasbacsai** [comment]
You just do not need to fill the FQDN and it will be basically a bot (only outgoing connections).

Is this not feasible for you?

**Reply by aidendotgg** [reply]
Yeah I guess that works. Still should be a separate option though so all of unrelated things are off of the dashboard
